src/assets/styles/tailwind.css

```
@@ -1,6 +1,19 @@
+
+@custom-variant dark (&:where(.dark, .dark *));
+
+@theme {
+  --color-primary: var(--aw-color-primary);
+  --color-secondary: var(--aw-color-secondary);
+  --color-accent: var(--aw-color-accent);
+  --color-default: var(--aw-color-text-default);
+  --color-muted: var(--aw-color-text-muted);
+
+  --font-sans: var(--aw-font-sans), ui-sans-serif, system-ui, sans-serif;
+  --font-serif: var(--aw-font-serif), ui-serif, Georgia, serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)